site/content/integrate/apps/quick-start-serverless/_index.md

```diff
@@ -0,0 +1,443 @@
+---
+title: "Quick start guide (serverless)"
+heading: "Writing a Mattermost app for AWS and OpenFaaS, in Go"
+description: "This quick start guide will walk you through the basics of writing a Mattermost app that can be deployed to serverless platforms, in Go."
+weight: 15
+---
+
+This quick start guide explains the basics of writing a Mattermost app that can
+be deployed to serverless platforms like AWS Lambda and OpenFaaS. In this guide
+you will build an app that contains:
+
+- [`manifest.json`](#1-manifest-manifestjson), declares itself an application
+  that can be deployed as HTTP, AWS Lambda, or OpenFaaS, that acts as a bot, and
+  attaches to locations in the user interface.
+- [`static/`](#2-static-static) with static assets.
+- [`function/`](#3-function-package) package that implements most of the app,
+  and is ready to be deployed to OpenFaaS. [`bindings.json`](#bindingsjson) to
+  attach the [`send_form.json`](#send_formjson) form to the `/hello-serverless
+  send` command. [`send()`](#functiongo) go function sends a message back to the
+  user.
+- [`aws/`](#4-aws) folder with `main.go` for AWS Lambda.
+- [`http/`](#5-http) folder with `main.go` to run a standalone HTTP server.
+- [`openfaas/`](#6-openfaas) folder with `manifest.yml` to deploy as an OpenFaaS function, and
+  a `golang-middleware` template.
+- [`Makefile`](#7-makefile) that can build aws and openfaas bundles (`make dist`).
+
+See complete source code of the example [here](https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-serverless). 
+
+## Prerequisites
+
+Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+
+In the System Console, ensure that the following are set to **true**:
+
+- `Enable Bot Account Creation`
+- `Enable OAuth 2.0 Service Provider`
+
+**Note:** Apps do not work with a production release of Mattermost right now. They can only be run in a development environment. A future release will support production environments.
```

Comment:
That is not longer the case, right?

Reply:
Thanks for fixing that for serverless. Would you mind also removing this section for the other two quick start guides?